frontend/components/player/playing_controls.jsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { GrCirclePlay } from "react-icons/gr";
+import { BiSkipPrevious, BiSkipNext, BiShuffle } from "react-icons/bi";
+import { BsRepeat, BsRepeat1, BsPauseCircle } from "react-icons/bs";
+import { useEffect } from "react";
+
+const PlayingControls = ({
+	isPlaying,
+	trackProgress,
+	togglePlay,
+	toPrevTrack,
+	toNextTrack,
+	toggleShuffle,
+}) => {
+	useEffect(() => {}, [isPlaying]);
```

Comment:
What does this do?

Reply:
Thanks, you're correct. I don't need this since isPlaying is being passed as a prop, so the component should re-render already based on that prop.